src/goto-programs/class_hierarchy.cpp

```diff
@@ -99,3 +104,8 @@ void class_hierarchyt::output(std::ostream &out) const
           << ch << '\n';
   }
 }
+
+bool class_hierarchyt::is_abstract_class(const irep_idt &c) const
+{
+  return abstract_classes.count(c) > 0;
```

Comment:
I understand that using `count` requires less typing, but then it does require looking up the container type to figure out what's going on. Would you mind using `abstract_classes.find(c) != abstract_classes.end()` instead?